.github/workflows/release.yml

```diff
@@ -228,18 +275,30 @@ jobs:
           ziti_ci_gpg_key_id: ${{ secrets.ZITI_CI_GPG_KEY_ID }}
         shell: bash
         run: |
+
+          set -o xtrace
+
           $(go env GOPATH)/bin/ziti-ci configure-git
-          $(go env GOPATH)/bin/ziti-ci publish-to-github -t --prerelease --archive-base "" --dry-run
-          mkdir tmp
-          tar xfzv ./release/ziti-linux-amd64-* -C ./tmp
-          $(go env GOPATH)/bin/ziti-ci verify-current-version -t $(./tmp/ziti version)
-          $(go env GOPATH)/bin/ziti-ci publish-to-github -t --prerelease --archive-base ""
+
+          $(go env GOPATH)/bin/ziti-ci publish-to-github --use-current-tag --prerelease --archive-base "" --dry-run
+
+          tar xfzv ./release/ziti-linux-amd64-* -C /tmp
+          # TODO: should this compare current tag with "ziti --version" instead?
```

Comment:
@plorenz I remember you mentioned somewhere that `ziti --version` was, in fact, different from `ziti version`. Are both still valid? If so, is `ziti version` still the correct variant for this case?